src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md

```diff
@@ -127,6 +127,15 @@ services.AddHttpClient(
 Note: The single instance returned by `HttpClientFactory` is reused by all
 export requests.
 
+## Troubleshooting
+
```

Comment:
I'll wait for more feedbacks before merging. The goal is to copy this to all the components, so want to make sure the content is fine :)